ch_backup/clickhouse/disks.py

```diff
@@ -187,9 +190,14 @@ def _copy_dir(
                 to_path,
             ]
 
+        ch_disks_config_path = "/tmp/clickhouse-disks-config.xml"
+        self._render_disks_config(
+            ch_disks_config_path,
+            {from_disk: self._disks[from_disk], to_disk: self._disks[to_disk]},
+        )
```

Comment:
It seems we will re-generate the config file for each invocation of `_copy_dir`. Suggest to move to `ClickHouseTemporaryDisks.__enter__`.

Reply:
Yes, it will re-generate but in `_render_disks_config` we will create config only for two disk: dest and source, instead of creating single config with all available disk. RN `clickhouse-disk` will load all disk from the config (https://github.com/ClickHouse/ClickHouse/issues/56791#issue-1994799242). Imo better to prevent unnecessary storage creating.